SCHIZO/Creatures/Ermshark/Ermshark.cs

```diff
@@ -12,6 +12,12 @@ public void OnTakeDamage(DamageInfo damageInfo)
     {
         if (liveMixin.health > 0) return;
 
+        if (damageInfo.dealer != Player.main.gameObject)
```

Comment:
Does this work for all damage types a player can cause? Propulsion Cannon, repulsion cannon, knife, thermoblade, ramming with seamoth, etc?

Reply:
Only the knives set the damage dealer - I think it's funnier if you can only kill the shark by stabbing but I can probably get vehicle collisions working too (lmk)